<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prism</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/archivos/archivos/Prism.png</normaloff>:/archivos/archivos/Prism.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>CÃ³digo</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>408</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="positionLabel">
          <property name="text">
           <string>Linea: 1, Col: 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="maximumSize">
           <size>
            <width>101</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Analizar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Consola</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>138</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="consola"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Tabla de Simbolos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Nombre</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tipo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valor</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionAbrir"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionGuardar_Como"/>
   <addaction name="actionCerrar"/>
   <addaction name="separator"/>
   <addaction name="actionCopiar"/>
   <addaction name="actionCortar"/>
   <addaction name="actionPegar"/>
   <addaction name="separator"/>
   <addaction name="actionAnalizar"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAYuda">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_Como"/>
    <addaction name="actionCerrar"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionCopiar"/>
    <addaction name="actionCortar"/>
    <addaction name="actionPegar"/>
   </widget>
   <addaction name="menuAYuda"/>
   <addaction name="menuEditar"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/nuevo.png</normaloff>:/archivos/archivos/nuevo.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/abrir.png</normaloff>:/archivos/archivos/abrir.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/guardar.png</normaloff>:/archivos/archivos/guardar.png</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/cerrar.png</normaloff>:/archivos/archivos/cerrar.png</iconset>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/copiar.png</normaloff>:/archivos/archivos/copiar.png</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/cortar.png</normaloff>:/archivos/archivos/cortar.png</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/pegar.png</normaloff>:/archivos/archivos/pegar.png</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/guardarcomo.png</normaloff>:/archivos/archivos/guardarcomo.png</iconset>
   </property>
   <property name="text">
    <string>Guardar Como</string>
   </property>
  </action>
  <action name="actionAnalizar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/archivos/archivos/Prism.png</normaloff>:/archivos/archivos/Prism.png</iconset>
   </property>
   <property name="text">
    <string>Analizar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
